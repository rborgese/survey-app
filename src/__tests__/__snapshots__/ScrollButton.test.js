import React from 'react';
import renderer from 'react-test-renderer'
import ScrollButton from '../../components/ScrollButton'

// should render the scroll button
test('scroll button renders correctly', () => {
    const component = renderer.create(<ScrollButton  scrollStepInPx="50" delayInMs="16.66"/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})

