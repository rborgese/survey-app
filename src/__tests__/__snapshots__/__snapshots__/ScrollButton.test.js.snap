// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scroll button renders correctly 1`] = `
<button
  className="scroll"
  onClick={[Function]}
  title="Back to top"
>
  <span
    className="arrow-up glyphicon glyphicon-chevron-up"
  />
</button>
`;
